tree/ntuple/v7/doc/specifications.md

```diff
@@ -395,6 +395,14 @@ If `flag==0x02` (_projected field_) is set,
 the field has been created as a virtual field from another, non-projected source field.
 If a projected field has attached columns,
 these columns are alias columns to physical columns attached to the source field.
+The following restrictions apply on field projections:
+  - The source field and the target field must have the same structure,
+    except for an `RNTupleCardinality` field that must have a collection field as a source.
```

Comment:
I'm not sure I fully understand the details here, but if I'm not mistaken for example the projected `RVec`s of the importer do not have the same subfield structure than the collections they point into. Is this covered here?

Reply:
clarified "structure" --> "structural role"